pkg/azuredisk/controllerserver.go

```diff
@@ -205,6 +209,20 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		}
 	}
 
+	diskName = path.Base(diskURI)
```

Comment:
is it ok to add a standalone func to check existence and capacity of azure disk?